cmd/config_test.go

```diff
@@ -98,3 +98,30 @@ func TestTLSConfigLoad(t *testing.T) {
 		})
 	}
 }
+
+func TestDefaultDBConfig(t *testing.T) {
+	exampleDBConfig := &DBConfig{}
+	exampleDatabaseConfig := &DatabaseConfig{
+		DBConfig{
+			DBConnect:       "some secret",
+			DBConnectFile:   "foo/bar",
+			MaxOpenConns:    100,
+			MaxIdleConns:    100,
+			ConnMaxLifetime: ConfigDuration{10000},
+			ConnMaxIdleTime: ConfigDuration{10000},
+		},
+	}
+	DefaultDBConfig(exampleDBConfig, exampleDatabaseConfig)
+	test.AssertEquals(t, *exampleDBConfig, exampleDatabaseConfig.DBConfig)
```

Comment:
I think this, like line 126, should just be asserting about the values within the resulting struct.